### DISCUSSION TITLE
Canvas Packaging: Any change to app yaml file causes parsing error

### DISCUSSION THREAD

**Discussion by MichaelJata** [microsoft/PowerApps-Language-Tooling#780] [Q&A]
Hi,
whenever I make any changes to e.g. App.fx.yaml, I get the following error when trying to pack it into a msapp:
`pac canvas pack --msapp Naacda530-e381-4650-82bc-2d6099c37a17-document.msapp --sources .\MSAPP-Content\`
**`PA3003: Parse error: Property should be at same indent level`**

As an example, in the OnStart event, I am setting up some Branding Colors and changed the variable from  "Black: ColorValue("#00000")" to "Black: ColorValue("#000000")". So just added "0" to the color value which works when changing in the App Studio.

After saving and executing the canvas pack, I am facing the mentioned error.
Just packing the package after unpacking (and without any modifications) works.

I am using Version: 1.29.11+g9e2b163

Any ideas about what is going wrong here?

**Comment by seanGSISG** [comment]
You ever figure this out?  Man from the future is having same issue.

Version: 1.42.1+g430e09e

**Comment by velocifer-in** [comment]
+1

**Comment by joem-msft** [comment]
Please create an issue if you need support.
And without knowing the content of the offending yaml file, it's hard to proceed. There will be some additional changes available in Pac CLI which will have some better error messages (e.g. including the file name in the error).
From the error message, it appears there's some invalid yaml files. I would consider opening all your files in VS Code, and see if there are any yaml problems first.
